packages/extension/snippets/emd.code-snippets

```diff
@@ -421,5 +421,15 @@
 			"<BaseMap>\n\t<Areas data={${1:query_name}} geoJsonUrl='path/to/geoJson' geoId=geoJsonField areaCol=column_name/>\n\t<Points data={${1:query_name}} lat=lat long=long/>\n</BaseMap>"
 		],
 		"description": "Insert Map"
+	},
+	"Note": {
+		"prefix": "/Note",
+		"body": ["<Note>\n\t${1:Note Content}\n</Note>\n\n"],
+		"description": "Insert Note"
+	},
+	"Embed": {
+		"prefix": "/Embed",
+		"body": ["<Embed url={${1:query_name}}/>\n\n"],
+		"description": "Insert Embed"
 	}
```

Comment:
Does this need to be updated for image and and link?

Reply:
Image and link already have markdown versions of them. Maybe we'll want to add them in the future, but I think it's okay for now